src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -93,6 +93,7 @@ public static class ScriptConstants
         public const string AntaresColdStartHeaderName = "X-MS-COLDSTART";
         public const string SiteTokenHeaderName = "x-ms-site-restricted-token";
         public const string EasyAuthIdentityHeader = "x-ms-client-principal";
+        public const string BackgroundSyncHeader = "x-ms-functions-client-backgroundsync";
```

Comment:
rename to `x-ms-functions-client-backgroundsynctrigger`

Reply:
Hi @pragnagopa the name is suggested by @balag0  that is reside on the resolved commend. :)